manifests/ta.pp

```diff
@@ -0,0 +1,70 @@
+#
+# Defined type: splunk::ta
+#
+# This define sets up a TA (Technology Addon) for Splunk.  It (optionally)
+# installed a package, and configures input forwarders in 
+# $SPLUNK_HOME/etc/apps/<app name>/local/inputs.conf
+#
+# Examples
+#
+# splunk::ta { 'search':
+#   install_package => false,
+# }
+#
+# splunk::ta::input { 'monitor:///var/log/messages':
+#   attributes => {
+#     'index' => 'server_t',
+#   },
+# }
+#
+# Alternatively you can feed inputs directly into splunk::ta using the
+# inputs parameter (useful if you are configuring from Hiera)
+#
+#
+# splunk::ta { 'search':
+#   install_package => false,
+#   inputs          => {
+#     'monitor:///var/log/messages' => {
+#       'attributes' => {
+#         'index' => 'server_t',
+#       }
+#     }
+#   }
+# }
+#
+define splunk::ta (
```

Comment:
Would this be easier understood if it was named `splunk::addon` or something? "ta" isn't something I would know without reading docs.